utils/index.d.ts

```diff
@@ -2100,3 +2123,223 @@ export declare namespace validationUtils {
      */
     export function getInvalidCharLengthMessage(constraints?: RangeConstraints): string;
 }
+
+// #region agent input types
+
+export type ParameterAgentMetadata = {
+    /**
+     * A camel cased string that names the parameter this quick pick is for. Will be given to a LLM to identify the
+     * parameter that it needs to pick a value for.
+     *
+     * For example:
+     * - "subscription"
+     * - "resourceGroup"
+     * - "runtime"
+     * - "name"
+     */
+    parameterName: string;
+
+    /**
+     * A title cased string for the parameter this quick pick is for. Will be displayed to the user.
+     *
+     * For example:
+     * - "Subscription"
+     * - "Resource Group"
+     * - "Runtime"
+     * - "Name"
+     */
+    parameterDisplayTitle: string;
+
+    /**
+     * A description of the parameter this quick pick is for. Will be displayed to the user.
+     *
+     * For example:
+     * - "The subscription that the Storage Account should be created in."
+     * - "The resource group that the Container App should be created in."
+     * - "The function runtime for the Function App."
+     * - "The name of the Static Web App."
+     */
+    parameterDisplayDescription: string;
+};
+
+export type AgentQuickPickItem<T extends QuickPickItem = QuickPickItem> = {
+    agentMetadata: {
```

Comment:
Could you add examples to the docs for these two properties?

Reply:
Yes.